### PR TITLE
MTM-51772 revoke tokens to log out users

### PR DESCRIPTION
**Pull request by mmed-softwareag**

### CODE REVIEW

File: content/users-guide/administration-bundle/managing-users.md [hunk 1]
```diff
@@ -181,12 +181,14 @@ Click the menu icon at the right of the respective row and then click **Delete**
...
 
-### To revoke user tokens
+### To log out all users
 
 In the event of a security incident involving the session tokens of your tenant's users, you can invalidate any tokens currently in use.
 
-To revoke all session tokens click **Revoke tokens**  at the right of the top menu bar. Revoking all tokens logs out all users logged in via "OAI-Secure" or "Single sign-on redirect". Note that JWT tokens retrieved by your devices will also be revoked.
+To log out all users click **Log out all users**  at the right of the top menu bar. This will log out all users logged with "OAI-Secure" or "Single sign-on redirect". JWT tokens retrieved by all devices in the current tenant will also be invalidated.
+
+Note that, if the "Basic Authentication" is used, the users logged with base64 token will logged out.
```
Comment (Michal-Bak) [inline content/users-guide/administration-bundle/managing-users.md:190]: I guess it is not true.

**Review by Michal-Bak — CHANGES_REQUESTED** [review]

File: content/users-guide/administration-bundle/managing-users.md [hunk 2]
```diff
@@ -181,12 +181,14 @@ Click the menu icon at the right of the respective row and then click **Delete**
 You need a role with user management permission to perform this option.
 {{< /c8y-admon-info >}}
 
-### To revoke user tokens
+### To log out all users
 
 In the event of a security incident involving the session tokens of your tenant's users, you can invalidate any tokens currently in use.
 
-To revoke all session tokens click **Revoke tokens**  at the right of the top menu bar. Revoking all tokens logs out all users logged in via "OAI-Secure" or "Single sign-on redirect". Note that JWT tokens retrieved by your devices will also be revoked.
+To log out all users click **Log out all users**  at the right of the top menu bar. This will log out all users logged with "OAI-Secure" or "Single sign-on redirect". JWT tokens retrieved by all devices in the current tenant will also be invalidated.
```
Comment (rssag) [inline content/users-guide/administration-bundle/managing-users.md:188]:
```suggestion
To log out all users click **Log out all users**  at the right of the top menu bar. This logs out all users currently logged in via OAI-Secure or single sign-on redirect. JWT tokens retrieved by all devices in the current tenant are also invalidated.
```

File: content/users-guide/administration-bundle/managing-users.md (hunk 1 above)
Comment (rssag) [inline content/users-guide/administration-bundle/managing-users.md:190]:
```suggestion
Note that, if basic authentication is used, users logged in via base64 token are logged out.
```

Comment (rssag) [inline content/users-guide/administration-bundle/managing-users.md:190]:
```suggestion
Note that, if the "Basic Authentication" is used, the users logged with base64 token are logged out.
```

**Review by Michal-Bak — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**